scripts/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+echo "=== START OF FILE ==="
+
+echo "==== <Check to see if user is root> ===="
+if [ $(whoami) != 'root' ]; then
+	echo "Please run this script as root or use sudo"
+	exit
+fi
+echo "==== </Check to see if user is root> ===="
+echo
+
+echo "==== <Set path to Bassa directory> ===="
+path=$(find ~  -name "Bassa")
+echo $path
+cd $path
+echo "==== </Set path to Bassa directory> ===="
+echo
+
+echo "==== <Python change 3.5> ===="
+pythonVersion=$(python --version | grep -o "[[:digit:]]\.[[:digit:]]")
+if [ $pythonVersion = "3.5" ]; then
+   echo "Verified that you are using python 3.5"
+else
+   rm /usr/bin/python
+   ln -s /usr/bin/python3.5 /usr/bin/python
```

Comment:
This might cause problems else where in the system. Wouldn't it be better if we remember the version and use it to run the scripts